extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/MessageBundleProcessor.java

```diff
@@ -552,7 +560,8 @@ private String generateImplementation(ClassInfo defaultBundleInterface, String d
 
             if (messageAnnotation == null) {
                 throw new MessageBundleException(
-                        "A message bundle interface method must be annotated with @Message: " + method);
+                        String.format("A message bundle interface method must be annotated with @Message on %s: ",
```

Comment:
I think there's a `%s` missing here or the method has one too many parameters.

Reply:
Good catch!